### DISCUSSION TITLE
migrate data between bitwarden and vaultwarden

### DISCUSSION THREAD

**Discussion by Blotz** [dani-garcia/vaultwarden#2717] [Q&A] [answered]
Is there a way that I can transfer my data(SQLite/vaultwarden) to a selfhosted bitwarden instance without exporting and importing my data manually?

**Comment by BlackDex** [comment, marked as answer]
To be short, no there isn't.

The main reason is both databases are formatted very differently, and there isn't a way (currently) to migrate that in a way understood by mssql and the way Bitwarden stores all the data.

I would suggest to use something like portwarden, that supports attachments also. Though, it doesn't support organizations very well if I'm correct.

**Reply by Blotz** [reply]
Ah thanks!
